project1/styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+    background-color: rgb(255, 220, 249);
+    font-family: 'Cute Font', cursive;
+    font-family: 'Lato', sans-serif;
+    font-family: 'Roboto', sans-serif;
+
+}
+
+* {
+    box-sizing: border-box;
+}
+
+main {
+    /* border: 10px solid #444; */
+
+    width: 1310px;
```

Comment:
Note that you set your width here to be _wider_ than the dimensions I said I would use for the project (1200px) in the requirements here: https://mica-web.github.io/gd-220/proj1.html